### ISSUE TITLE
Title of event is not shown in agenda view in mobile view

### ISSUE THREAD

**Issue by vmario89** [niccokunzmann/open-web-calendar#277]
Hi, the following little bug appears on mobile view. seems it happens when title is too long

![Screenshot_20240218-225518_Firefox](https://github.com/niccokunzmann/open-web-calendar/assets/5631071/5c37a507-1418-49a6-a812-876cf90ad129)
![Screenshot_20240218-225524_Firefox](https://github.com/niccokunzmann/open-web-calendar/assets/5631071/a6901040-5b65-4996-a141-4590b8079cb8)

<small>
<hr/>
We're using <a href="https://polar.sh/niccokunzmann">Polar.sh</a> so you can upvote and help fund this issue. We receive the funding once the issue is completed & confirmed by you. Thank you in advance for helping prioritize & fund our work.
</small>

<a href="https://polar.sh/niccokunzmann/open-web-calendar/issues/277">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/277/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/277/pledge.svg">
</picture>
</a>

**Comment by vmario89** [comment]
a fix for this in css is:

```
div.dhx_agenda_line > span {
	display: block ruby !important;
	text-overflow: ellipsis;
	overflow: hidden;
}
```

![grafik](https://github.com/niccokunzmann/open-web-calendar/assets/5631071/60a78ade-c571-403b-aa89-4d6edfbd1161)

**Comment by niccokunzmann** [comment]
@vmario89 Thanks for reporting this - would you like to create a Pull Request to add this to the style.css file?
https://github.com/niccokunzmann/open-web-calendar/blob/master/static/css/dhtmlx/style.css

**Comment by niccokunzmann** [comment]
fixed in #281
